coresdk/src/test/test_input.cpp

```diff
@@ -139,3 +158,6 @@ void run_input_test()
     
     free_all_fonts();
 }
+
+
+```
```

Comment:
Is the ``` meant to be here?

Reply:
That's not supposed to be there, good spot.